src/klocek.cpp

```diff
@@ -60,6 +60,23 @@ std::vector<std::vector<bool>> Klocek::dajTablice()
 }
 std::vector<std::vector<bool>> Klocek::obroc(KierunekObrotu kierunekObrotu = PRAWO)
 {
-    // TODO
-    return {{}};
+    if(kierunekObrotu == LEWO){
+        for(int i = 0 ; i < 2; i++){
+            obroc();
+        }
+    }
+
+    std::vector<std::vector<bool>> nowa_tablica = tablica;
+
+    for(int i = 0; i < wielkosc; i++){
+        for(int j = 0 ; j < wielkosc; j++){
+            int k = j;
+            int l = wielkosc - i - 1;
+            nowa_tablica[k][l] = tablica[i][j];
```

Comment:
Deklarowanie zmiennych jest niepotrzebne osobno, można to zrobić już w nawiasach kwadratowych.